source/type/type.h

```diff
@@ -1021,9 +1019,11 @@ struct MetaVehicleJourney: public Header, HasMessages {
     get_vj_at_date(RTLevel level, const boost::gregorian::date& date) const;
     std::vector<VehicleJourney*>
     get_vjs_in_period(RTLevel level, const boost::gregorian::date_period& period) const;
-
 private:
-    navitia::flat_enum_map<RTLevel, std::vector<VehicleJourney*>> rtlevel_to_vjs_map;
+    template<typename VJ>
+    VJ* set_vj(std::unique_ptr<VJ>& vj_ptr, RTLevel level);
+private:
```

Comment:
I'm used to have a private section for members and a private section for functions... but ok, i'll remove it